lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,154 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone',
+        'js/student_account/models/account_settings_models',
+        'js/student_account/views/account_settings_fields',
+        'js/student_account/views/account_settings_view',
+    ], function (gettext, $, _, Backbone, AccountSettingsModel, AccountSettingsFieldViews, AccountSettingsView) {
+
+        var setupAccountSettingsSection = function (fields_data) {
+
+            var accountSettingsElement = $('.account-settings-container');
+            var accountSettingsModel = new AccountSettingsModel();
+            accountSettingsModel.url = accountSettingsElement.data('accounts-api-url');
+
+            var accountSettingsView = new AccountSettingsView({
+                el: accountSettingsElement,
+                model: accountSettingsModel,
+                context: {
+                    accountSettingsModelUrl: accountSettingsModel.url
+                },
+                sections: [
+                    {
+                        title: gettext("Basic Account Information"),
+                        fields: [
+                            {
+                                view: new AccountSettingsFieldViews.ReadonlyFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Username"),
+                                    valueAttribute: "username",
+                                    helpMessage: "",
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.TextFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Full Name"),
+                                    valueAttribute: "name",
+                                    helpMessage: gettext("This is used on your edX certificates, and all changes are reviewed."),
```

Comment:
I'm not sure what "changes are reviewed" means here - who does the reviewing? edX? The student? Also, what would that person be checking for - illegal characters, for example?